src/desktop/apps/pro_buyer/pages/landing/client/view.coffee

```diff
@@ -63,9 +63,11 @@ module.exports = class ProfessionalBuyerLandingView extends View
   authenticate: (mode) ->
     mediator.trigger 'open:auth',
       mode: mode
-      width: '500px'
       copy: 'Artsy Professional Buyer Program'
       redirectTo: "#{SHOW_PATH}/complete"
+      trigger: 'click'
+      intent: mode
```

Comment:
Not sure how the mode correlates to the intent here? Should it not be something about professional buyer program?

Reply:
Ah yeah, good catch. I think I did this because an `intent` for PB doesn't exist in our schema doc. And also, I've been asked to complete this ticket before I leave: https://artsyproduct.atlassian.net/browse/GROW-752

Reply:
Ah makes sense!